src/__tests__/time.spec.js

```diff
@@ -0,0 +1,21 @@
+import { formatTime } from "../SoundList";
+
+describe("format time", () => {
+  it("should format time", () => {
+    expect(formatTime(40271)).toEqual("11:11:11");
+    expect(formatTime(36000)).toEqual("10:00:00");
+    expect(formatTime(35999)).toEqual("09:59:59");
+    expect(formatTime(3661)).toEqual("01:01:01");
+    expect(formatTime(3661)).toEqual("01:01:01");
+    expect(formatTime(3601)).toEqual("01:00:01");
+    expect(formatTime(3600)).toEqual("01:00:00");
+    expect(formatTime(3599)).toEqual("00:59:59");
+    expect(formatTime(601)).toEqual("00:10:01");
+    expect(formatTime(600)).toEqual("00:10:00");
+    expect(formatTime(599)).toEqual("00:09:59");
+    expect(formatTime(61)).toEqual("00:01:01");
+    expect(formatTime(60)).toEqual("00:01:00");
+    expect(formatTime(59)).toEqual("00:00:59");
+    expect(formatTime(0)).toEqual("00:00:00");
```

Comment:
Most sounds don't use hrs formatting. Here's the expectation i think we should have for this function:

```js
expect(formatTime(0)).toEqual("00:00"); // if hrs === 0, do not show it
expect(formatTime(40271)).toEqual("11:11:11"); // if hrs > 0, do show it
```

Reply:
I was considering that earlier, but I thought it we might want uniform sound formats. I removed the hours' place of sounds' durations if they're less than an hour in length with the latest commit at the time of this writing.

Removing the hours' place of the sound duration creates a somewhat large gap in the sound data icons, so I also gave the icon denoting the number of downloads a bit more space in the latest commit at the time of this writing.